app/controllers/concerns/blacklight/default_component_configuration.rb

```diff
@@ -64,39 +65,26 @@ def add_show_tools_partial name, opts = {}
         end
       end
 
-      ##
       # Add a tool to be displayed for each document in the search results.
-      # @param partial [String] the name of the document partial
-      # @param opts [Hash]
-      # @option opts [Symbol,Proc] :if render this action if the method identified by the symbol or the proc evaluates to true.
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      # @option opts [Symbol,Proc] :unless render this action unless the method identified by the symbol or the proc evaluates to true
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      def add_results_document_tool name, opts = {}
+      # @see #add_nav_action for params
```

Comment:
No, it just points to it.  @see could also reference external URLs.  They happen to share the same args, because they are consumed in the same environments and the opts hash is common to all of them.  A macro might be more useful if we actually cared about generating standalone YARD/rdoc.  I don't.  I expect to read the docs in the code.